packages/dashmate/docker-compose.yml

```diff
@@ -50,7 +50,7 @@ services:
       - ${PLATFORM_DRIVE_ABCI_LOG_JSON_DIRECTORY_PATH:?err}:/var/log/dash/drive/json
     environment:
       - BLOCK_SPACING_MS=3000 # TODO: sync with tenderdash
-      - CHAIN_ID=devnet # TODO: sync with tenderdash chain id
+      - CHAIN_ID=${PLATFORM_DRIVE_TENDERDASH_GENESIS_CHAIN_ID:-devnet}
```

Comment:
when we release to mainnet, will this break?

Reply:
No, it would break previously. This change simply takes chain ID from tenderdash config.

But I changed it to error when empty.